docker/Dockerfile

```diff
@@ -57,9 +23,6 @@ USER 1000
 
 ENV PATH=/opensearch-benchmark/venv/bin:$PATH
```

Comment:
Is this correct?  I would assume `pip` would install OSB into the user (or system library) and there would be no `venv` in this case?

Reply:
This is correcet. When I remove the line, it doesn't change anything else except for when I check `which opensearch-benchmark`, it preappends `opensearch-benchmark/venv/bin:/` to the `$PATH`. 
```
$ echo $PATH
/opensearch-benchmark/venv/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```